.github/workflows/linux-build.yml

```
@@ -32,12 +31,28 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
```

I recommend to use `v3` of the checkout action to suppress a github warning.

```suggestion
        uses: actions/checkout@v3
```